static/scss/layout.scss

```diff
@@ -1,12 +1,15 @@
+@import "variables";
 @import "node_modules/@material/layout-grid/dist/mdc.layout-grid";
 @import "node_modules/@material/top-app-bar/dist/mdc.top-app-bar";
+@import "node_modules/bootstrap/scss/bootstrap";
 @import "common";
 @import "top-app-bar";
 @import "collapsible";
 @import "catalog";
 @import "checkout";
+@import "course_details/header";
 
 body {
-  font-family: "Open Sans","Helvetica Neue",Helvetica,Arial,sans-serif;
+  font-family: "Rajdhani", "Open Sans","Helvetica Neue",Helvetica,Arial,sans-serif;
```

Comment:
I'm guessing this is here to make it easier for your team to develop these pages? Not too important, but what happens for you guys if you move `"Rajdhani"` after `Arial` in this list? If your text still shows up normally, let's make that change.

Reply:
@gsidebo `Rajdhani` is a non-standard font and by the time the browser can get to it (after Arial) we already have a font that is available on the system. Moving it after Arial results in Open Sans displaying on Linux and Helvetiva Neue on MacOS.

Reply:
Rajdhani is the font spec'ed in the design, right?

Reply:
@pdpinch Yes, that's the font used in the PSDs.